<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-U73VDTJ" failures="1" tests="8" name="Default test" time="0.455" errors="0" timestamp="2025-02-28T19:43:45 IST">
  <testcase classname="Testngpackage.Sampletest" name="Dependencymethod" time="0.063">
    <failure type="java.lang.Error" message="Unresolved compilation problem: 
	sample cannot be resolved to a variable
">
      <![CDATA[java.lang.Error: Unresolved compilation problem: 
sample cannot be resolved to a variable

at Testngpackage.Sampletest.Dependencymethod(Sampletest.java:39)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Dependencymethod -->
  <testcase classname="Testngpackage.Sampletest" name="samplemethod" time="0.012"/>
  <testcase classname="Testngpackage.Sampletest" name="samplemethod" time="0.005"/>
  <testcase classname="Testngpackage.Sampletest" name="samplemethod" time="0.004"/>
  <testcase classname="Testngpackage.Sampletest" name="samplemethod" time="0.015"/>
  <testcase classname="Testngpackage.Sampletest" name="samplemethod" time="0.002"/>
  <testcase classname="Testngpackage.Sampletest" name="samplemethod2" time="0.016"/>
  <testcase classname="Testngpackage.Sampletest" name="samplemethod1" time="0.003"/>
  <testcase classname="Testngpackage.Sampletest" name="samplemethod3" time="0.0">
    <ignored/>
  </testcase> <!-- samplemethod3 -->
</testsuite> <!-- Default test -->
